### PR TITLE
Add automatic releases to deploy.yml

### LINKED ISSUE #104: Automatic releases
We want to use a automatic creation of releases on push to master. The description should just use the pull request name and description

### PR DESCRIPTION
**Pull request by Jakobis**
Solves #104  I'm not sure how to test this except committing it. Any advice on how to do this would be appreciated.
I decided to stick with automatic naming for now, and if it's bad, then I'll look into a better way of doing it.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -38,3 +38,6 @@ jobs:
                       cd devoops
                       git pull
                       ./deploy.sh
+            - name: GH Release
+              uses: softprops/action-gh-release@v0.1.5
```
Comment (jlndk) [inline .github/workflows/deploy.yml:42]: According to the action you're using (https://github.com/softprops/action-gh-release) the GITHUB_TOKEN is required.
```suggestion
              uses: softprops/action-gh-release@v0.1.5
              env:
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Review by jlndk — DISMISSED** [review]
Seems like a good solution.

I however wonder if it is possible to get a changelog automaticly or if that is out of scope for this PR?

**Comment by Jakobis** [comment]
> Seems like a good solution.
> 
> I however wonder if it is possible to get a changelog automaticly or if that is out of scope for this PR?

Probably slightly out of scope, I can take of it next week, but for now I still need to make sure that SonarCloud works, so I'd prefer to focus on that.

**Comment by Hjaltesorgenfrei** [comment]
Cant we just add from here https://github.com/softprops/action-gh-release:
```
      - name: Generate Changelog
        run: echo "# Good things have arrived" > ${{ github.workflow }}-CHANGELOG.txt
      - name: Release
        uses: softprops/action-gh-release@v1
        if: startsWith(github.ref, 'refs/tags/')
        with:
          body_path: ${{ github.workflow }}-CHANGELOG.txt
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Review by Hjaltesorgenfrei — APPROVED** [review]

**Pull request merged**